junit/src/main/java/cucumber/runtime/junit/SkippedThrowable.java

```diff
@@ -0,0 +1,13 @@
+package cucumber.runtime.junit;
+
+class SkippedThrowable extends Throwable {
+    private static final long serialVersionUID = 1L;
+
+    public SkippedThrowable() {
+        this(NotificationLevel.STEP);
+    }
+
+    public SkippedThrowable(NotificationLevel scenarioOrStep) {
+        super(String.format("This %s is skipped", scenarioOrStep.lowerCaseName()), null, false, false);
```

Comment:
Since v1.1.6 ["JDK7 is required to build everything. Built jars should still work on JDK6" (see History.md)](https://github.com/cucumber/cucumber-jvm/blob/master/History.md#116-2014-03-24). I would be a good idea to make it explicit in the pom.